src/vumps_localham.jl

```diff
@@ -402,17 +415,30 @@ function tdvp_iteration_parallel(
     Ãᶜ[n] = Avecsₙ[1]
   end
 
-  # TODO: based on minimum singular values of C̃, use more accurate
-  # method for finding Ãᴸ, Ãᴿ
+  for n in 1:Nsites
+    Cvecₙ = updater(Hᶜ(∑h, Hᴸ, Hᴿ, ψ, n),ψ.C[n])
+    Avecₙ = updater(Hᴬᶜ(∑h, Hᴸ, Hᴿ, ψ, n), ψ.AL[n] * ψ.C[n])
+    C̃[n] = Cvecₙ
+    Ãᶜ[n] = Avecₙ
```

Comment:
This should get updated with the new `solver` syntax (pretty much the same, but with a few more inputs and outputs).